src/connectors/cloudtrail.py

```diff
@@ -0,0 +1,168 @@
+from runners.helpers import db
+from runners.helpers.dbconfig import WAREHOUSE
+
+CONNECTION_OPTIONS = {
```

Comment:
so either making a manual list in `__init__.py` or having it list the directory and check the modules for `CONNECTION_OPTIONS` would help the `GET /data/connectors` endpoint along here

Reply:
A list of CONNECTION_OPTIONS for all of the connectors?